slides/index.html

```diff
@@ -20,11 +20,23 @@
 
 ## Hvem er vi?
 
-Studerer datateknikk på NTNU
+???
+
+- Først og fremst takk til fagkom for gratis muffins og kaffe
```

Comment:
Also, we will be using these slides with tihlde, too. So we'll have to remember to update this part